generator/routing_index_generator.cpp

```diff
@@ -272,11 +317,20 @@ void CalcCrossMwmTransitions(string const & path, string const & mwmFile, string
       CrossMwmConnectorSerializer::AddTransition(transition, mask, connectors[i]);
     }
   }
+
+  for (size_t i = 0; i < connectors.size(); ++i)
+  {
+    auto const vehicleType = static_cast<VehicleType>(i);
+    auto const & connector = connectors[i];
+    LOG(LINFO, (vehicleType, "model: enters:", connector.GetEnters().size(), ", exits:",
```

Comment:
num enters, num exits

Reply:
`LOG(LINFO, (vehicleType, "model. Number of enters:", connector.GetEnters().size(), "Number of exits:", connector.GetExits().size()));` Ok?